### PR TITLE
Improve travis test runs, this should also speed things up a bit.

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

**Comment by diox** [comment]
Are you sure removing the parameter to build in parallel is a good idea ? It seems to make things slower.

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --reuse-db --instafail --showlocals
```
Comment (diox) [inline setup.cfg]: Remove instafail from requirements as well.

Comment (EnTeQuAk) [inline setup.cfg]: Ah, right. done

**Comment by EnTeQuAk** [comment]
yeah it seems you're right, I reverted the paralel change and let's see how that goes.

**Comment by diox** [comment]
I think the number of concurrent processes to use was determined by trial and error. I suspect it will stay faster than alternatives as long as travis don't change anything on their side.

**Comment by diox** [comment]
You could reference https://github.com/mozilla/addons-server/issues/2228 and maybe implement more ideas from that. In particular, removing the need to fetch `product_details` in the `assets` build should gain us a minute or two.

**Comment by EnTeQuAk** [comment]
Damn, I didn't see that ticket. I'll see what we can do, thanks for linking this!

**Comment by eviljeff** [comment]
Is there anything that can be landed from this?  Seems like we try to fix everything, find problems, so end up committing nothing so travis is still slow.

**Comment by EnTeQuAk** [comment]
I think we could just merge this PR and work on the other ideas one after another.

so, r? :sweat_smile:

File: .travis.yml [hunk 2]
```diff
@@ -11,22 +13,38 @@ env:
...
   - scripts/travis_es.sh
   - /tmp/elasticsearch/elasticsearch-1.6.2/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
+
 install:
   - nvm current
   - nvm deactivate
   - nvm install 4
   - nvm use 4
   - pip install tox==1.8.1
+  - pip install --upgrade pip wheel
```
Comment (diox) [inline .travis.yml]: do that before tox.

Comment (EnTeQuAk) [inline .travis.yml]: fixed

**Comment by diox** [comment]
r+

**Pull request merged**